{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "bf1c7bde_a8fa28f1",
        "filename": "vpd-manager/src/backup_restore.cpp",
        "patchSetId": 1
      },
      "lineNbr": 28,
      "author": {
        "id": 1000522
      },
      "writtenOn": "2025-02-19T09:52:24Z",
      "side": 1,
      "message": "As we handled exception in the JSON utility, We don;t need to handle exception here again.",
      "revId": "c7499d4da695a63ff0491bae6f1c0fd2f412a8a5",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "59aeb292_9bb997df",
        "filename": "vpd-manager/src/backup_restore.cpp",
        "patchSetId": 1
      },
      "lineNbr": 28,
      "author": {
        "id": 1001845
      },
      "writtenOn": "2025-02-19T10:33:45Z",
      "side": 1,
      "message": "Done",
      "parentUuid": "bf1c7bde_a8fa28f1",
      "revId": "c7499d4da695a63ff0491bae6f1c0fd2f412a8a5",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "91b0dd3c_a9cef5a2",
        "filename": "vpd-manager/src/backup_restore.cpp",
        "patchSetId": 1
      },
      "lineNbr": 28,
      "author": {
        "id": 1000522
      },
      "writtenOn": "2025-02-20T03:22:38Z",
      "side": 1,
      "message": "Done",
      "parentUuid": "59aeb292_9bb997df",
      "revId": "c7499d4da695a63ff0491bae6f1c0fd2f412a8a5",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    }
  ]
}